Yoast/javascript
# Create a styled component from the Loader so we don't have to add scss.

## Summary
This PR creates a styled-component from our Loader so that we don't need any additional scss files for it.

## Relevant technical choices:

* Remove all the scss!

## Test instructions
This PR can be tested by following these steps:
* Make sure that you know how the loader should look like (for example, by checking the components app).
* Generate a new distributable css file by running `grunt build` in the `packages/yoast-components` directory. 
* Copy these files to the `apps/components/css-dist` directory.
* Restart the app and make sure that the Loader still looks the same.

Note: Don't look at the wizard with the updated `.css` files. Due to the changes in #141 this will look awful.
Note: If grunt is complaining about the old import of the `configuration-wizard.scss`, just remove the line from `all.scss`.

## UI changes
This PR SHOULD NOT change anything in the UI. If it does, please send it back.

## Quality assurance


Fixes #151
